core/kamon-core/src/main/scala/kamon/Kamon.scala

```diff
@@ -16,12 +16,31 @@
 
 package kamon
 
+import kamon.metric.MetricRegistry
+import kamon.trace.Tracer
+import kamon.util.Clock
+
+import java.util.concurrent.ScheduledExecutorService
+
+//class Kamon {
+//
+//
+//  def tracer: Tracer = ???
+//  def clock: Clock = ???
+//  def scheduler: ScheduledExecutorService = ???
+//  def metricRegistry: MetricRegistry = ???
+//
+//}
+
```

Comment:
Garbage?

Reply:
Yeah, this is still an early draft :smile: